tests/functional/test_changelog_basic_ansible.py

```diff
@@ -899,6 +900,7 @@ def test_changelog_release_ansible_plugin_cache(  # pylint: disable=redefined-ou
 notesdir: fragments
 prelude_section_name: release_summary
 new_plugins_after_name: removed_features
+add_plugin_period: true
```

Comment:
I would only set this to `true` for some tests, so that both code paths (with and without this option) are tested.
```suggestion
add_plugin_period: false
```